backend/src/routes/api/prometheus/index.ts

```diff
@@ -39,6 +40,19 @@ module.exports = async (fastify: KubeFastifyInstance) => {
     },
   );
 
+  fastify.post(
+    '/pvc-range',
```

Comment:
are you sure this is `pvc` and not just data from Thanos?

Reply:
Based on looking at the query -- you are not after PVC data...

PVC => Persistent Volume Claim; aka a cluster storage.

You'll need to name this as something else. Your data is just in Thanos / UWM (I think they are the same).

Can we rename `prometheusPVC` to `prometheusThanos` function and you call your endpoint something for your use case?

Reply:
Makes sense, can do!